__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const db = require('../db/connection.js');
+const app = require('../app');
+const request = require('supertest');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+beforeEach(() => {
+	return seed(testData);
+});
+afterAll(() => {
+	return db.end();
+});
+
+describe('topics', () => {
+	test('200 /api/topics responds with an array of topics', () => {
+		return request(app)
+			.get('/api/topics')
+			.expect(200)
+			.then((response) => {
+				expect(response.body.topics).toBeInstanceOf(Object);
+				response.body.topics.forEach((topic) => {
+					expect(topic).toMatchObject({
+						slug: expect.any(String),
+						description: expect.any(String),
```

Comment:
this test looks good, I would add one more assertion to check that the array is the correct length